node/lib/DeviceAccessor.ts

```diff
@@ -341,7 +347,15 @@ export class DeviceAccessor {
 
     private static capitalize(propertyName: string) {
         if (propertyName.length > 1) {
-            return propertyName.substr(0, 1).toUpperCase() + propertyName.substr(1);
+            return propertyName[0].toUpperCase() + propertyName.substr(1);
+        }
+
+        return propertyName;
+    }
+
+    private static uncapitalize(propertyName: string) {
+        if (propertyName.length > 1) {
+            return propertyName[0].toLowerCase() + propertyName.substr(1);
         }
```

Comment:
Or are you suggesting the method names are confusing?

---
In reply to: [73988719](https://github.com/openT2T/opent2t/pull/15#discussion_r73988719) [](ancestors = 73988719,73968450)